charms/kfp-persistence/src/charm.py

```diff
@@ -33,7 +33,7 @@
 log = logging.getLogger()
 
 K8S_RESOURCE_FILES = ["src/templates/auth_manifests.yaml.j2"]
-SA_NAME = "ml-pipeline-persistenceagent"
+SA_NAME = "kfp-persistence"
```

Comment:
question: Why do we need this change?

Reply:
actually, is the token bound to this service account? I don't see any such service account being created anywhere, right? How was it working previously?